// @chipi-pay/chipi-sdk/dist/index.js
'use strict';

var chunkDBK5U5WV_js = require('./chunk-DBK5U5WV.js');



Object.defineProperty(exports, "BACKEND_URL", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.BACKEND_URL; }
});
Object.defineProperty(exports, "ChipiProvider", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.ChipiProvider; }
});
Object.defineProperty(exports, "ChipiSDK", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.ChipiSDK; }
});
Object.defineProperty(exports, "createArgentWallet", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.createArgentWallet; }
});
Object.defineProperty(exports, "executePaymasterTransaction", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.executePaymasterTransaction; }
});
Object.defineProperty(exports, "useApprove", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useApprove; }
});
Object.defineProperty(exports, "useCallAnyContract", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useCallAnyContract; }
});
Object.defineProperty(exports, "useChipiContext", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useChipiContext; }
});
Object.defineProperty(exports, "useCreateWallet", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useCreateWallet; }
});
Object.defineProperty(exports, "useStakeVesuUsdc", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useStakeVesuUsdc; }
});
Object.defineProperty(exports, "useTransfer", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useTransfer; }
});
Object.defineProperty(exports, "useWithdrawVesuUsdc", {
  enumerable: true,
  get: function () { return chunkDBK5U5WV_js.useWithdrawVesuUsdc; }
});
